lib/KRadioButtonGroup.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+
+  <div class="k-radio-button-group">
+    <!-- @slot To render the KRadioButtons -->
+    <slot></slot>
+  </div>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'KRadioButtonGroup',
+    props: {
+      /**
+       * Specifies whether the radio button group is enabled or disabled.
+       * Used to make the first radio button active when the radio group becomes enabled again after being disabled.
+       */
+      enable: {
```

Comment:
> I don't quite understand yet in what case we'd need to use `enable` functionality, would you explain on an example?

https://github.com/user-attachments/assets/df3498f8-e6e0-40f5-858a-a0e5d114dfe7

It was a redundant logic. In the settings page earlier I used two KRadioButtonGroup, 

first to wrap **Learn-page** and **Sign-in page** options
second to wrap **below three radio buttons.**

What used to happen was after we had selected an option from those 3 radio buttons and when we chose **Learn-page** option that disables the **Sign-in page** options. Then when we again chose **Sign-in page** radio, it enables its options but instead of the first option being selected by default the last option which we have chosen before used to be selected. 
For that, I have used that **enable prop** logic. I check if the old enable value is false and the new enable value is true then select the first sign-in page option and the old option must not remain selected.

But now it does not happen because I use only a single KRadioButtonGroup component to wrap all these 5 radio buttons.

Reply:
I see, thanks for explanation. I think we can remove it then. If there are some situations when we need to focus the first radio button, I think we can use `KRadioButton`s public `focus` method for the same purpose.